scripts/buildRpm.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
@yannick-polius, check out the buildRpm.sh and packaging spec in NMProtobuffers to see an example of how to pull a specific branch from git for building an rpm. In NMProtobuffers, the Protobuffers repo is added into the NMProtobuffers build, so it's not exactly what you are looking for, but you'll see the following in the spec file:
<pre><code>
# Fetch the protofiles
/bin/echo $PWD
/bin/sh scripts/getProtofiles.sh %{version} %{protouser}
if [ ! -d "../Protobuffers" ]; then
   /bin/echo "Protobuffers branch %{protouser}/Protobuffers.git/%{version} not found.";
   exit
fi
</code></pre>